sys/zuse.hoon

```diff
@@ -7105,20 +7066,55 @@
       {$j gift:able:jael}
   ==
 ::
-++  unix-task                                           ::  input from unix
-  $%  {$belt p/belt:dill}                               ::  dill: keyboard
-      {$blew p/blew:dill}                               ::  dill: configure
-      {$boat ~}                                         ::  clay: reboot
-      {$born ~}                                         ::  eyre: new process
-      [%crud tag=@tas =tang]                            ::  any vane: error report
-      {$hail ~}                                         ::  dill: refresh
-      {$hear p/lane:ames q/@}                           ::  ames: input packet
-      {$hook ~}                                         ::  dill: hangup
-      {$into p/desk q/? r/mode:clay}                    ::  clay: external edit
-      {$they p/@ud q/httr:eyre}                         ::  eyre: in response
-      {$this p/? q/clip:eyre r/httq:eyre}               ::  eyre: in request
-      {$thud ~}                                         ::  eyre: in cancel
-      {$wake ~}                                         ::  behn: wakeup
++$  unix-task                                           ::  input from unix
```

Comment:
It might be nice to define vane interfaces in terms of this, rather than the other way around. That way you could look at a vane interface and immediately see which moves come from Unix. Could be done in a different PR, though.